pkg/storage/poststorage/redis_search.go

```diff
@@ -45,6 +45,70 @@ func (r *Redis) SearchComment(cla []objectid.ID) ([]*Object, error) {
 	return out, nil
 }
 
+func (r *Redis) SearchCreated(beg int, end int) ([]*Object, error) {
+	var err error
+
+	// val will result in a list of all post IDs within the given pagination
+	// range, if any.
+	var val []string
+	{
+		val, err = r.red.Sorted().Search().Order(storageformat.PostCreated, -(end + 1), -(beg + 1))
```

Comment:
The way Redis works is quite complex and takes some time to internalize. There is a lot of documentation about all the Redis commands and our interface documentation usually explains well what this is supposed to do here. Admittedly, this example here is one of the more complicated queries.